demo/js/demo.js

```diff
@@ -982,6 +963,16 @@ window.onload = function() {
                 // idx.SQUARE_OF_INCOME
             ],
 
+            backgroundColor: 'white',
+            gridRowProperties: [
+                undefined,
```

Comment:
I think it would be cleaner to have object with the row index as keys as opposed to a sparse array